src/utils/misc.ts

```diff
@@ -153,3 +154,66 @@ export const getUpdateAuthor: (id: string, isReviewer: boolean) => UpdateAuthor
   id,
   role: isReviewer ? DacoRole.ADMIN : DacoRole.SUBMITTER,
 });
+
+const appHistoryTSVColumns: ColumnHeader[] = [
+  { name: 'Application #', accessor: 'appId' },
+  {
+    name: 'Date of Status Change',
+    accessor: 'date',
+    format: (value: string) => moment(value).format('YYYY-MM-DD'),
+  },
+  { name: 'Application Status', accessor: 'eventType' },
+  { name: 'Application Type', accessor: 'appType' },
+  { name: 'Action Performed By', accessor: 'role' },
+  { name: 'Days Since Last Status Change', accessor: 'daysElapsed' },
+  { name: 'Institution', accessor: 'institution' },
+  { name: 'Country', accessor: 'country' },
+  { name: 'Applicant', accessor: 'applicant' },
+  { name: 'Project Title', accessor: 'projectTitle' },
+  { name: 'Ethics Letter', accessor: 'ethicsLetterRequired' },
+];
+
+const sortByDate = (a: any, b: any) => {
+  return b.date.getTime() - a.date.getTime();
+};
+
+export const createAppHistoryTSV = (results: ApplicationDocument[]) => {
```

Comment:
this function can be in the service, it contains logic to transform data, and would only be used in the service, my suggestion is to move it there